cpp/models/sde_seirvv/parameters.h

```diff
@@ -0,0 +1,303 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Nils Wassmuth, Rene Schmieding, Martin J. Kuehn
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef MIO_SDE_SEIRVV_PARAMETERS_H
+#define MIO_SDE_SEIRVV_PARAMETERS_H
+
+#include "memilio/utils/uncertain_value.h"
+#include "memilio/epidemiology/contact_matrix.h"
+#include "memilio/utils/parameter_set.h"
+
+namespace mio
+{
+namespace sseirvv
+{
+
+/***************************************
+ * Define Parameters of the SSEIRVV model *
+ ***************************************/
+
+/**
+ * @brief Probability of getting infected from a contact 
+ * with variant 1.
+ */
+struct TransmissionProbabilityOnContactV1 {
+    using Type = UncertainValue<>;
+    static Type get_default()
+    {
+        return Type(1.0);
+    }
+    static std::string name()
+    {
+        return "TransmissionProbabilityOnContactV1";
+    }
+};
+
+/**
+ * @brief Probability of getting infected from a contact 
+ * with variant 2.
+ */
+struct TransmissionProbabilityOnContactV2 {
+    using Type = UncertainValue<>;
+    static Type get_default()
+    {
+        return Type(1.0);
+    }
+    static std::string name()
+    {
+        return "TransmissionProbabilityOnContactV2";
+    }
+};
+
+/**
+ * @brief The latent time of variant 1 in day unit.
+ */
```

Comment:
```suggestion
/**
 * @brief The latent time in number of days of variant 1.
 */
```
You can use another expression, but "in day unit" does not sound like proper English to me. Please fix below as well.